components/template-editor.tsx

```diff
@@ -71,6 +88,47 @@ const PreviewComp: React.FC<{
   )
 }
 
+const AttachmentsComp: React.FC = () => {
+  const attachments = useTemplateStore((state) => state.attachments)
+  const remainingCount = Math.max(zenv.NEXT_PUBLIC_ATTACHMENT_MAX_FILES - attachments.length, 0)
+  return (
+    <>
+      {attachments.length > 0 && (
+        <Group wrap='wrap' gap='xs'>
+          {attachments.map((file, index) => (
+            <Pill
+              key={`${file.name}-${index}`}
+              withRemoveButton
+              onRemove={() => {
+                useTemplateStore.getState().removeAttachment(file)
+              }}
+            >
+              {file.name}
+            </Pill>
+          ))}
+        </Group>
+      )}
+      <Dropzone
+        onDrop={(files) => {
+          useTemplateStore.getState().addAttachments(files)
+        }}
+        maxFiles={remainingCount}
+        maxSize={zenv.NEXT_PUBLIC_ATTACHMENT_MAX_SIZE_MB * 1024 * 1024}
```

Comment:
Why is there a max of 3 attachments?

Reply:
> Why is there a max of 3 attachments?

We could set it in environment variables.  All data is sent through mime message, so I thought it might be good to limit the number of files and size per files.